scripts/parse.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+
+from jinja2 import Environment
+from jinja2.loaders import FileSystemLoader
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(
+        prog="AMS swagger parser",
+        description="Extracts AMS configuration data",
+    )
+    parser.add_argument(
+        "swagger_path",
+        help="Path to the swagger json file (Should conform to swagger spec 2.0)",
+    )
+    parser.add_argument(
+        "--output",
+        dest="output_file",
+        help="Destination of the rendered configuration file",
+        default="ams-configuration.md",
+    )
+    args = parser.parse_args()
+    with open(args.swagger_path, mode="r") as f:
+        swagger = json.load(f)
+
+    configs = _parse_config_schema(swagger)
+    nodes = _parse_node_schema(swagger)
+    env = Environment(loader=FileSystemLoader("."))
+    templ = env.get_template("template.md.j2")
+    text = templ.render(configs=configs, nodes=nodes)
+    with open(args.output_file, mode="w+") as op:
+        op.write(text)
+
+
+def _parse_config_schema(swagger) -> dict:
+    ams_config_path = "/1.0/config"
+    config_props = swagger["paths"][ams_config_path]["get"]["responses"]["200"][
```

Comment:
This looks a bit weird.

Reply:
in what way ?

Reply:
The following looks decent to me 
```
config_pros = swagger["paths"][ams_config_path]["get"]["responses"]["200"]
        ["schema"]["properties"]["metadata"]["properties"]["config"]["properties"]
```
rather than the way it looked like above

Reply:
That is just `black` formatting this with I think the character limit restrictions.

Reply:
TBH, that looks odd.